ddd-and-spring/index.adoc

```diff
@@ -58,7 +58,7 @@ The order context might know about a product through its line items but actually
 image::bounded-context1.png[]
 
 This boils down to model elements which might appear as a singular concept at first glance needing to be reflected in different way in different aspects of the system.
-Modern software architecture even sometimes takes this a step further and even aligns system boundaries with such a bounded context, actively accepts redundance and eventual consistency to reduce the complexity of the individual systems, foster resilience and increase development productivity.footnoteref:[microservices, Microservices - https://en.wikipedia.org/wiki/Microservices[Wikipedia]].
+Modern software architecture even sometimes takes this a step further and even aligns system boundaries with such a bounded context, accepts redundancy to create consistency which reduces the complexity of the individual systems, fosters resilience and increases development productivity.footnoteref:[microservices, Microservices - https://en.wikipedia.org/wiki/Microservices[Wikipedia]].
```

Comment:
I'm gonna keep "actively accepts redundance and eventual consistency to reduce the complexity" as your change alters the semantics.